9.4/root/usr/libexec/fix-permissions

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# Fix permissions on the given directory to allow group read/write of
+# regular files and execute of directories.
+chown -R postgres "$1"
+chgrp -R 0 "$1"
+chmod -R g+rw "$1"
+find "$1" -type d -exec chmod g+x {} +
```

Comment:
Are these commands allowed to fail? If not, we should add `set -e`. We should also make sure $1 is defined. Together with what we had previously, `set -eu` is ideal.